composer.json

```diff
@@ -12,14 +12,20 @@
         }
     ],
     "require": {
-        "php": "^5.4|^7.0"
+        "php": "^7.0",
+        "psr/http-client": "^1.0",
+        "psr/http-factory": "^1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
         "mockery/mockery": "~0.8",
-        "guzzlehttp/guzzle": "~5.0"
+        "guzzlehttp/guzzle": "~5.0",
+        "symfony/http-client": "^4.3",
+        "nyholm/psr7": "^1.1"
     },
     "suggest": {
+        "symfony/http-client": "Provide an implementation of PSR-18 HTTP client",
```

Comment:
let's remove, suggests line are noise anyway
maybe remove the line about guzzle instead?

Reply:
OK, no let's keep it (but alpha order + no mention of PSR18)